pkgs/development/python-modules/bcg/default.nix

```diff
@@ -0,0 +1,49 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, appdirs
+, click
+, click-log
+, paho-mqtt
+, pyaml
+, pyserial
+, schema
+, simplejson
+}:
+buildPythonPackage rec {
+  pname = "bcg";
+  version = "1.17.0";
+
+  propagatedBuildInputs = [
+    appdirs
+    click
+    click-log
+    paho-mqtt
+    pyaml
+    pyserial
+    schema
+    simplejson
+  ];
+
+  src = fetchFromGitHub {
+    owner = "hardwario";
+    repo = "bch-gateway";
+    rev = "v${version}";
+    sha256 = "2Yh5MeIv+BIxjoO9GOPqq7xTAFhyBvnxPy7DeO2FrkI=";
+  };
+
+  postPatch = ''
+    sed -ri 's/@@VERSION@@/${version}/g' \
+      bcg/__init__.py setup.py
+  '';
```

Comment:
we normally order these three like: src, postPatch, propagatedBuildInputs

Reply:
Ok.